src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1989,6 +1989,24 @@ limitations under the License.
             <include cluster="Operational State" client="false" server="true" clientLocked="true" serverLocked="true"></include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-robotic-vacuum-cleaner</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Robotic Vacuum Cleaner</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0074</deviceId>
+        <clusters lockOthers="true">
+            <include cluster="Identify" client="false" server="true" clientLocked="true" serverLocked="true">
+                <requireAttribute>IDENTIFY_TIME</requireAttribute>
+                <requireAttribute>IDENTIFY_TYPE</requireAttribute>
+                <requireCommand>Identify</requireCommand>
+            </include>
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true"></include>
+            <include cluster="RVC Run Mode" client="false" server="false" clientLocked="true" serverLocked="true"></include>
```

Comment:
```suggestion
            <include cluster="RVC Run Mode" client="false" server="true" clientLocked="true" serverLocked="true"></include>
```
because what's there now says "not enabled and can't be enabled"....